apps/myjobs/app/assets/javascripts/joyride.js

```diff
@@ -495,7 +495,7 @@
                     } else if (methods.top()) {
 
                         settings.$next_tip.css({
-                            top: (settings.$target.offset().top - settings.$next_tip.outerHeight() - nub_height + topAdjustment),
+                            top: settings.$target.offset().top + topAdjustment,
```

Comment:
It was suggested offline that modification to the library was no longer required. Can changes to this file then be undone?